R/rstan_config.R

```diff
@@ -179,9 +179,9 @@ rstan_config <- function(pkgdir = ".") {
   if (grepl("\\.stanfunctions$", file_name) &&
       (utils::packageVersion('rstan') < 2.29)) {
     mod <- readLines(file_name)
-    file_name <- paste0(.basename_noext(file_name), "_wrapped.stanfunctions")
-    cat("functions {", mod, "}", sep = "\n",
-        file = file_name)
+    if (!any(grepl("\\bfunctions \\{", mod))) {
+      cat("functions {", mod, "}", sep = "\n", file = file_name)
```

Comment:
CRAN wants things like this to be done with `writeLines` rather than `cat` and I guess not everyone includes a space between functions and the {

Reply:
Good call! Fixed